addon/models/github-organization.js

```diff
@@ -1,12 +1,23 @@
 import Model from 'ember-data/model';
 import attr from 'ember-data/attr';
 import { hasMany } from 'ember-data/relationships';
+import { deprecate } from '@ember/application/deprecations';
+import { computed } from '@ember/object';
 
 export default Model.extend({
   login: attr('string'),
   name: attr('string'),
   avatarUrl: attr('string'),
 
-  members: hasMany('github-member', { inverse: null }),
-  repositories: hasMany('github-repository')
+  members: hasMany('github-member'),
+  repositories: hasMany('github-repository'),
+
+  githubUsers: computed(function() {
```

Comment:
Does this need to depend on `members` in case the data in the model changes, so it can recompute? Same question for repos below.

Reply:
Good point, it probably does. I’ll update all of these to account for that.

Reply:
Updated